core-contracts/AssetManager/src/main/java/network/balanced/score/core/asset/manager/AssetManagerImpl.java

```diff
@@ -95,6 +98,17 @@ public Address getAddress(String name) {
         return BalancedAddressManager.getAddressByName(name);
     }
 
+    @External
+    public void setAssetChainDepositLimit(Address asset, String nid, BigInteger limit) {
+        onlyGovernance();
+        assetChainDepositLimit.at(asset).set(nid, limit);
```

Comment:
so i think this should change to 
```suggestion
    public void setAssetChainDepositLimit(String tokenNetworkAddress, BigInteger limit) {
        onlyGovernance();
        assetChainDepositLimit.set(tokenNetworkAddress, limit);
```